src/test/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/integration/ApplicationDocumentsTest.kt

```diff
@@ -78,64 +77,39 @@ class ApplicationDocumentsTest : InitialiseDatabasePerClassTestBase() {
           .expectBody()
           .json(
             objectMapper.writeValueAsString(
-              documentTransformer.transformToApiUnfiltered(groupedDocuments),
+              documentTransformer.transformToApi(groupedDocuments),
             ),
           )
       }
     }
   }
 
   @Test
-  fun `Get application documents for Temporary Accommodation returns 200`() {
```

Comment:
As mentioned in PR description, David A and I, discovered that the 'Temporary Accommodation' flow is no longer used so the related business-logic has also been stripped out as well as this test to cover that case